packages/client/src/modules/administration/Games/Game/Animation/Launch.tsx

```diff
@@ -26,17 +23,55 @@ const hasTeamWithTooManyPlayers = (
   );
 };
 
+const hasPlayersWithoutTeam = (teams: ITeamWithPlayers[]) => {
+  const noTeamPlayers = teams.find(
+    (team: ITeamWithPlayers) => (team.name = NO_TEAM)
+  )?.players;
+  return noTeamPlayers && noTeamPlayers?.length > 0;
+};
+
+const hasPlayersWithUnvalidatedForm = (players: any[], gameId: number) => {
+  return players
+    .map(({ playedGames }) => {
+      const currentGame = playedGames.find(
+        ({ gameId: gId }: { gameId: number }) => gId === gameId
+      );
+      return currentGame?.profile?.status === "pendingValidation";
+    })
+    .filter(Boolean).length;
+};
+
 export default function Launch({ game }: { game: IGameWithTeams }) {
   const queryClient = useQueryClient();
-  const [open, setOpen] = useState(false);
-  const handleClickOpen = () => setOpen(true);
-  const handleClose = () => setOpen(false);
+  const [errorDialogOpen, setErrorDialogOpen] = useState(false);
+  const [successDialogOpen, setSuccessDialogOpen] = useState(false);
   const MAX_TEAM_SIZE = 5;
   const dialogContent = {
     message: "Êtes-vous sûr.e de vouloir lancer l'atelier ?",
     warningMessage: `Une ou plusieurs équipe(s) dépasse(nt) le nombre de joueurs recommandé (${MAX_TEAM_SIZE} joueurs)`,
   };
 
+  const playersQuery = usePlayers(game.id);
+  const players =
+    playersQuery?.data?.data?.players?.map((p: any) => ({
+      id: p.id,
+      playedGames: p.playedGames,
+    })) ?? [];
+  const playersWithUnvalidatedForms = hasPlayersWithUnvalidatedForm(
+    players,
+    game.id
+  );
+
+  const playersWithoutTeams = hasPlayersWithoutTeam(game.teams);
+
+  const errorDialog = {
```

Comment:
Les messages peuvent être placés dans le fichier de traduction i18n

Reply:
Par rapport au message avec une condition à l'intérieur, est ce que je crée deux messages dans le fichier et j'appelle en fonction de la condition ? 
Ou je fais une nouvelle méthode qui intègre la condition et je met avec la condition dans le fichier de traductions ? 
(c'est juste pour un singulier / pluriel mais bon on aura sûrement la question dans d'autres cas)

Reply:
Bonne question. Pour le moment c'est un système custom donc vaut mieux garder un fichier json simple, sans fonction dedans : du coup 2 clés de traduction.
Il faut qu'à terme on utilise la lib i18n qui permet de gérer les cas singulier/pluriel out of ze box